src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -11,12 +11,19 @@
 public class ProgrammingLife {
 
     private static final String DATA_FOLDER = "data/";
+    private static final String TEST_DATA = DATA_FOLDER + "test/test.gfa";
     private static final String TB_DATA = DATA_FOLDER + "real/TB10.gfa";
     private static final String HUMAN_DATA = DATA_FOLDER + "real/chr19.hg38.w115.gfa";
 
     public static void main(String[] args) {
+        Graph g;
+        String graphFile = TEST_DATA;
+
+        if (args.length > 0)
+            graphFile = args[0];
+
         try {
-            Graph g = Graph.parse(HUMAN_DATA);
+            g = Graph.parse(HUMAN_DATA);
         } catch (FileNotFoundException e) {
             e.printStackTrace();
```

Comment:
Please give an actually useful error message instead of printing the stacktrace.